core/chains/evm/config/toml/defaults/Shibarium_Testnet.toml

```diff
@@ -0,0 +1,38 @@
+ChainID = '157'
+FinalityTagEnabled = true
+LinkContractAddress = '0x44637eEfD71A090990f89faEC7022fc74B2969aD'
+LogPollInterval = '1s'
+MinIncomingConfirmations = 5
+NoNewHeadsThreshold = '30s'
+RPCBlockQueryDelay = 10
+RPCDefaultBatchSize = 100
+
+[Transactions]
+ResendAfterThreshold = '1m'
+
+[BalanceMonitor]
+Enabled = true
+
+[GasEstimator]
+EIP1559DynamicFees = true
+PriceDefault = '10 gwei'
+PriceMin = '2 gwei'
```

Comment:
Same here as mainnet. Think these 2 configs aren't really doing anything since eip1559 is enabled.